src/components/ButtonGroup/ButtonGroup.tsx

```diff
@@ -39,25 +39,9 @@ export const ButtonGroup = chakra(
       ...rest
     } = props;
     const newChildren: JSX.Element[] = [];
-    // Based on --nypl-breakpoint-medium
-    const breakpointMedium = 600;
-    const [finalLayout, setFinalLayout] = React.useState<LayoutTypes>(layout);
-    const [finalButtonWidth, setFinalButtonWidth] =
-      React.useState<ButtonGroupWidths>(buttonWidth);
-    const windowDimensions = useWindowSize();
-    React.useEffect(() => {
-      // When on a mobile device or narrow window, always set the layout to
-      // column and the button width to "full".
-      if (windowDimensions.width <= breakpointMedium) {
-        setFinalButtonWidth("full");
-        setFinalLayout("column");
-      } else {
-        // Otherwise, set the layout and button width to the values
-        // passed in via the `buttonWidth` and `layout` props.
-        setFinalButtonWidth(buttonWidth);
-        setFinalLayout(layout);
-      }
-    }, [buttonWidth, layout, windowDimensions.width]);
+    const { isLargerThanMedium } = useNYPLBreakpoints();
```

Comment:
`isLargerThanMedium` is used three times right here in this file.  And I am guessing that is predominantly what I am going to see throughout the rest of the code changes, as that is the breakpoint we target for mobile-to-desktop UI format changes.  Would it make sense to also include something like `isLargerThanMobile`?